packages/website/.eslintrc.json

```diff
@@ -0,0 +1,12 @@
+{
+  "extends": ["plugin:react/recommended"],
+  "rules": {
+    "react/prop-types": "off",
```

Comment:
This should not be necessary since we're using TypeScript. The compiler can type-check properties